.github/ISSUE_TEMPLATE/onboarding_argocd.yaml

```diff
@@ -0,0 +1,101 @@
+          options:
+              - Zero
+              - Infra
+              - Rick
+    - type: input
+      id: managed-namespaces
+      attributes:
+          label: Managed namespaces
+          description: The target namespaces that should be managed by ArgoCD. Preferably using [a single (team name) prefix](https://github.com/operate-first/support/blob/main/docs/onboarding_to_argocd.md#additional-notes).
+          placeholder: target-namespace
+    - type: markdown
+      attributes:
+          value: "# Deploying a new application"
+    - type: input
+      id: project-name
+      attributes:
+          label: Team name or your Argo CD project
+          description: Please specify your Argo CD project name. If you're using this template to request both, project and application, you can skip this field.
+          placeholder: argocd-project-name
+    - type: input
+      id: git-source
+      attributes:
+          label: Application source git repository
+          description: URL to the source repository. Please specify if the repository is private. In that case we'll exchange credentials later on.
+          placeholder: repository-url
+    - type: input
+      id: manifest-path
+      attributes:
+          label: Path to manifests within the repository
+          description: Relative path within the repository to the resources you want Argo CD to deploy.
+          placeholder: relative-path
+    - type: dropdown
+      id: target-cluster-new-app
+      attributes:
+          label: Target cluster
+          description: Choose an environment from [this list](https://github.com/operate-first/apps/tree/master/cluster-scope/overlays)
+          options:
+              - Zero
+              - Infra
+              - Rick
+    - type: input
+      id: target-namespace-new-app
+      attributes:
+          label: Target namespace for the application
+          description: Please specify an existing namespace in the target cluster which is owned by your team. If you want a new namespace to be created, please file an issue via the [Onboarding to a cluster](https://github.com/operate-first/support/issues/new/choose) template.
+          placeholder: my-namespace
+    - type: markdown
+      attributes:
+          value: "# Add a new user to an ArgoCD Project"
+    - type: input
+      id: new-username
+      attributes:
+          label: User name
+          description: Please specify your MOC OCP User Name.
+          placeholder: john at gmail.com
+    - type: input
+      id: new-user-team-name
+      attributes:
+          label: Team name
+          description: Please specify your team Name.
+          placeholder: my-team-name
+    - type: input
+      id: new-user-argocd-project
+      attributes:
+          label: ArgoCD Project
+          description: Please specify the ArgoCD project this user should be added to.
+          placeholder: ArgoCD project
```

Comment:
can you change `ArgoCD project` to `argocd-project-name` to be consistent with your other place holder formatting.